### ISSUE TITLE
Not finding HK path and not allowing manual path choice

### ISSUE THREAD

**Issue by lorefield** [fifty-six/Scarab#177]
### Scarab Version

v1.34.0.0

### Hollow Knight Version

v1.5.78.11833

### Steps to Reproduce

I don't know.

### Expected Behaviour

Setting the installation path or warning that an invalid path was selected.

### Actual Behaviour

When I launch Scarab it tells me 'Unable to detect your Hollow Knight installation. Please select it.'. When I navigate to the install folder (in my case: /opt/steam/steamapps/common/Hollow Knight/) clicking 'Open' on the location picker dialog does nothing. (N.B. Clicking 'Open'  while a file in the installation folder is selected will close the dialog with 'no path was selected')

### Logs

This is all the log after starting Scarab, interactions (e.g. clicking 'open') with the location dialog don't create more log entries.

```
Launching...
LogHost: Initializing to normal mode (.cctor)
Checking if up to date...
Loading settings.
[Binding]Error in binding to 'Avalonia.Input.KeyBinding'.'Command': 'Null value in expression '{empty}' at ''.'(KeyBinding #18307385)
[Binding]Error in binding to 'Avalonia.Input.KeyBinding'.'Command': 'Null value in expression '{empty}' at ''.'(KeyBinding #30548742)
[Binding]Error in binding to 'Avalonia.Input.KeyBinding'.'Command': 'Null value in expression '{empty}' at ''.'(KeyBinding #6503226)
```

**Comment by fifty-six** [comment]
Ah sorry I forgot to respond to this, could I see the output of tree `/opt/steam/steamapps/common/Hollow Knight`? (If you haven't already figured this out)

**Comment by lorefield** [comment]
[hktree.txt](https://github.com/fifty-six/Scarab/files/12340289/hktree.txt)

**Comment by fifty-six** [comment]
Hey, sorry for a late response again, I've been messing with the path stuff in the meantime - I'm thinking of adding an option to either use the avalonia file picker or just let you choose the executable. I was also gonna ask, what distro/de/etc are you using? I haven't been able to repro this locally yet, but someone else has had the same issue

**Comment by lorefield** [comment]
My turn to say, "woops late response" ;-)

Debian 10.13 / KDE Plasma 5.14.5 / don't know what else the etc would be

**Comment by fifty-six** [comment]
Should be fixed with v2.4!
